lib/point_quest_web/live/quest.ex

```diff
@@ -439,6 +439,23 @@ defmodule PointQuestWeb.QuestLive do
     {:noreply, assign(socket, attacks: attacks)}
   end
 
+  # remove from quest if you were just booted
+  def handle_info(
+        %Event.AdventurerRemovedFromParty{adventurer_id: adventurer_id},
+        %{assigns: %{actor: %Actor.Adventurer{adventurer: %{id: adventurer_id}}}} = socket
+      ) do
+    socket = put_flash(socket, :info, "your bitch ass was removed")
```

Comment:
This put_flash is ignored/vestigial for now, since they end up on an error page when they complete the redirect, which will not display or persist the flash message.